pulp_smash/tests/docker/api_v2/test_crud.py

```diff
@@ -15,6 +15,11 @@
 from pulp_smash.constants import REPOSITORY_PATH
 
 
+def setUpModule():  # pylint:disable=invalid-name
```

Comment:
I'm wondering if there is a way to avoid repeat this code over the other modules. Not sure if an import from other module will help.